modules/tinymce/src/core/main/ts/api/Options.ts

```diff
@@ -369,6 +369,15 @@ const register = (editor: Editor) => {
     processor: 'string'
   });
 
+  registerOption('skin', {
+    processor: (value) => Type.isString(value) || value === false,
+    default: 'oxide'
+  });
+
+  registerOption('skin_url', {
+    processor: 'string'
+  });
```

Comment:
Forgot to mention this when I logged the PR but I was tossing up between whether this should be in Core or in the Silver Theme. Would be keen to get others thoughts?

Reply:
Ooh, tough choice. It depends on whether the skin setting is ever read in core (other than to download the skin, although perhaps that counts).

The skin is not useful without the theme, so I suggest putting the registration in the theme, unless that would cause problems with downloading the skin.

Reply:
The skin option I don't believe is ever read in core, as it's loaded by the theme. The reason I placed it in core is it's used by other plugins (e.g. template, preview and exports)

Reply:
Anyways yeah I really just flipped a coin here... I do agree thinking more on this that it should be in the theme, as each theme may have a different default skin.